core/src/main/resources/error/error-classes.json

```
@@ -757,6 +757,21 @@
       "The expression <sqlExpr> cannot be used as a grouping expression because its data type <dataType> is not an orderable data type."
     ]
   },
+  "HLL_INVALID_INPUT_SKETCH_BUFFER" : {
+    "message" : [
+      "Invalid call to <function>; only valid HLL sketch buffers are supported as inputs (such as those produced by the HLL_SKETCH_AGG function)."
```

```suggestion
      "Invalid call to <function>; only valid HLL sketch buffers are supported as inputs (such as those produced by the `hll_sketch_agg` expression)."
```
The functions/expression names are all lowercase (see that `prettyName` string for the expressions).

---

Done.